packages/plugins/src/hardhat/simulate.ts

```diff
@@ -249,6 +260,39 @@ export const setupPresimulationState = async (
   return signer
 }
 
+/**
+ * Fetches the transaction response objects for all of the transactions in the simulation. There's some additional
+ * data in the responses that is not in the receipts (data, value, etc) which is not available in the receipts. We
+ * use this information to calculate deployment cost estimates on the website.
+ */
+export const fetchTransactionResponses = async (
```

Comment:
Does this function actually return the blob gas used given that the transactions are executed on a Hardhat node?

Reply:
I'm not entirely sure, I was planning to test this out once the upgrade went live on Optimism (which just happened today).

Reply:
Yeah, unfortunately, the blob gas is not calculated during the simulation, so I'm just going to remove all the changes related to it.